src/components/home/blank_slate.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const BlankSlate = () => {
+  return (
+    <div className="blank-slate">
+      <span role="img" aria-label="sunglass" className="blank-slate__emoji">
+        😎
```

Comment:
@noor-ahmed How about creating a chore task to change this emoji to an SVG icon? 💭

Reply:
@carryall created here [task-19](https://github.com/noor-ahmed/nimble-survey/projects/1#card-52972135)